scripts/babel_update.sh

```diff
@@ -45,7 +45,7 @@ pybabel extract \
   --sort-output \
   --copyright-holder=Superset \
   --project=Superset \
-  -k _ -k __ -k t -k tn -k tct .
+  -k _ -k __ -k t -k tn:1,2 -k tct .
```

Comment:
Bycatch - we probably don't need `-k tct` here as I don't believe we even have such a translator function, but I suppose it doesn't matter that much.